core/src/main/res/values-qq/strings.xml

```diff
@@ -42,4 +42,7 @@
   <string name="search_history">TODO: Unclear, must be documented. See https://github.com/kiwix/overview/issues/31</string>
   <string name="save">{{Identical|Save}}</string>
   <string name="no_bookmarks">This means \"there are no bookmarks\"</string>
+  <string name="on">{{Identical|Turn On}}</string>
```

Comment:
@dipu989 I haven't actually edited values-qq myself yet.

@Abijeet could I bother you to chime in on how this should look?

Reply:
@Abijeet will these addition to `qq` provide value to the translators?

Reply:
Not really, it would be more useful to have a string like this, just an example,

> This is used in the settings screen to turn on notification.

Reply:
Or rather looking at this particularly,

> This is used in the settings screen to turn on night mode.

Reply:
@dipu989 if you can make the updates to the remaining qq strings we can close off this pr.

Thank you @Abijeet